config/regtest.json5

```
@@ -0,0 +1,11 @@
+{
+  app: {
+    devMode: true,
+    checkInterval: 10,
+    logLevel: 5,
+    workdir: './tmp/server'
```

```suggestion
    workdir: '/tmp/server'
```
I'd use the `/tmp` folder for this purpose instead of having a local folder being created for this (otherwise I guess we need to add it in the `.gitignore` file.

---

done